app.js

```diff
@@ -1,15 +1,20 @@
 const express = require("express");
 const { getTopics } = require("./controllers/topics.controllers");
-const { handleServerErrors, handleFourOhFour } = require("./errors/errors");
+const { handleServerErrors, handleFourOhFour, handlePsqlErrors, handleCustomErrors } = require("./errors/errors");
+const { getArticleById } = require("./controllers/articles.controllers");
 
 const app = express();
 
 app.use(express.json());
```

Comment:
Please delete app.use(express.json()) and only add it in when it is relevant to the task, (it's for dealing with requests that have a body)